pkg/skaffold/debug/debug_test.go

```diff
@@ -114,14 +115,14 @@ func (t testTransformer) RuntimeSupportImage() string {
 	return ""
 }
 
-func (t testTransformer) Apply(container *v1.Container, config imageConfiguration, portAlloc portAllocator) map[string]interface{} {
+func (t testTransformer) Apply(container *v1.Container, config imageConfiguration, portAlloc portAllocator) *ContainerDebugConfiguration {
```

Comment:
Can you return a value rather than a pointer?

Reply:
I think this is a vestige prior to introducing `IsApplicable()`.  I'll do that as a separate PR if that's ok.